scripts/build-upload-a-docker-image.sh

```diff
@@ -68,4 +68,6 @@ docker buildx build --output "${PUSHTAG}" \
 	${IMAGE_TAGS} \
 	${dir_arg}
 
+bash update-dockerhub-description.sh ${IMAGE_TAGS} ${dir_arg}
```

Comment:
```suggestion
bash update-dockerhub-description.sh "${IMAGE_TAGS}" ${dir_arg}
```
IMAGE_TAGS may contain all kinds of things, do you actually need them all, or do you only need namespace/component?

Reply:
I have updated it and the final command in build-upload-a-docker-image.sh looks like:

`bash update-dockerhub-description.sh ${component_name} ${dir_arg}/README.md`

`namespace="jaegertracing"` is already covered in the update-dockerhub-description.sh

The first argument taken is the image_name and the second argument is the readme_path